src/main/java/reposense/model/WizardCliArguments.java

```diff
@@ -0,0 +1,7 @@
+package reposense.model;
+
+/**
+ * Represents command line arguments user supplied when running the program with mandatory field -init.
```

Comment:
--init

Reply:
I am following the other similar files for documentation consistency such as `LocationsCliArguments` and `ViewCliArguments`, where they chose to use a single dash.

Reply:
Oh I see, didn't realize that!